pkg/auth/dependency/user/store.go

```diff
@@ -113,16 +116,24 @@ func (s *Store) UpdateMetadata(userID string, metadata map[string]interface{}, u
 	return nil
 }
 
-func (s *Store) UpdateLoginTime(userID string, lastLoginAt time.Time) error {
+func (s *Store) UpdateLoginTime(userID string, loginAt time.Time) (*time.Time, error) {
 	builder := s.SQLBuilder.Tenant().
 		Update(s.SQLBuilder.FullTableName("user")).
-		Set("last_login_at", lastLoginAt).
-		Where("id = ?", userID)
+		Set("last_login_at", squirrel.Expr("login_at")).
+		Set("login_at", loginAt).
+		Where("id = ?", userID).
+		Suffix("RETURNING last_login_at")
```

Comment:
I think we should add LoginAt and LastLoginAt to User. This function keeps returning error only. The caller calls Get to retrieve the updated user.